Task3/Project/DataControl/Services/CsvFileService.cs

```diff
@@ -1,53 +1,346 @@
-using DataControl.Interfaces;
+using System.IO;
 
 namespace DataControl.Services
 {
-    public class CsvFileService : IDataAccessService
+    /// <summary>
+    /// Represents service for work with csv files.
+    /// </summary>
+    public class CsvFileService : Interfaces.IDataAccessService
     {
+        // FIELDS
+        private TaxiDriver.Driver driver;
+        private string message;
+        private FileConfiguration fileConfiguration;
+
+        // PROPERTIES
+        /// <summary>
+        /// Current driver.
+        /// </summary>
         public TaxiDriver.Driver Driver
         {
             get
             {
-                throw new System.NotImplementedException();
+                return driver;
             }
         }
-
+        /// <summary>
+        /// Some special information about exceptional service state.
+        /// </summary>
         public string Message
         {
             get
             {
-                throw new System.NotImplementedException();
+                return message;
+            }
+        }
+
+        // METHODS
+        private TaxiDriver.Client CreateClient()
+        {
+            string[] clientLines;
+            using (StreamReader streamReader = new StreamReader(fileConfiguration.ClientFile))
+            {
+                clientLines = streamReader.ReadToEnd().Trim().Split('\n');
+            }
+            System.Random rand = new System.Random();
+            int clientIndex = rand.Next(0, clientLines.Length);
+
+            string[] clientParameters = clientLines[clientIndex].Split(';');
+            return new TaxiDriver.Client(clientParameters[1], clientParameters[clientParameters.Length - 1]);
+        }
+
+        private TaxiDriver.Route CreateRoute()
+        {
+            string[] routeLines;
+            using (StreamReader streamReader = new StreamReader(fileConfiguration.RouteFile))
+            {
+                routeLines = streamReader.ReadToEnd().Trim().Split('\n');
+            }
+            System.Random rand = new System.Random();
+            int routeIndex = rand.Next(0, routeLines.Length);
+
+            string[] routeParameters = routeLines[routeIndex].Split(';');
+
+            int startStreetId = int.Parse(routeParameters[1]);
+            int endStreetId = int.Parse(routeParameters[2]);
+
+            System.Tuple<string, string> startAndEndStreets = GetStartAndEndStreets(startStreetId, endStreetId);
+
+            return new TaxiDriver.Route(startAndEndStreets.Item1,
+                startAndEndStreets.Item2,
+                new System.TimeSpan(0, 0, int.Parse(routeParameters[3])),

```

Comment:
TimeSpan.FromSecond or some other method, check it

Reply:
yes, thanks